dags/resources/stages/parse/table_definitions/idex/idex_event_Cancel.json

```diff
@@ -0,0 +1,121 @@
+{
+    "parser": {
+        "type": "log",
+        "contract_address": "0x2a0c0dbecc7e4d658f48e01e3fa353f44050c208",
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": false,
+                    "name": "tokenBuy",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "name": "amountBuy",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "name": "tokenSell",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "name": "amountSell",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "name": "expires",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "name": "nonce",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "name": "user",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "name": "v",
+                    "type": "uint8"
+                },
+                {
+                    "indexed": false,
+                    "name": "r",
+                    "type": "bytes32"
+                },
+                {
+                    "indexed": false,
+                    "name": "s",
+                    "type": "bytes32"
+                }
+            ],
+            "name": "Cancel",
+            "type": "event"
+        },
+        "field_mapping": {}
+    },
+    "table": {
+        "dataset_name": "idex",
+        "table_name": "idex_event_Cancel",
```

Comment:
Please change the name to Exchange_event_Cancel for consistency with other datasets. The 1st part is the contract name which can be found here https://etherscan.io/address/0x2a0c0dbecc7e4d658f48e01e3fa353f44050c208#code after "Contract Name: ".

Reply:
Done!